src/types/s3.js

```diff
@@ -25,6 +25,7 @@ export type S3Bucket = {|
 
 export type S3Object = {|
     +Key: string,
+    +name: string,
```

Comment:
Why do we need `name` here?

Reply:
done: Not at all, very good point. I did a mistaking using S3Object type instead Object type for testing Objects components, i just remove it !